ksp_plugin/part.hpp

```diff
@@ -170,10 +173,14 @@ class Part final {
        RigidMotion<RigidPart, Barycentric> rigid_motion,
        std::function<void()> deletion_callback);
 
+  static RigidMotion<RigidPart, EccentricPart> ToEccentric(
```

Comment:
1. Please comment this function.
1. I find the name confusing.  In particular at the call site I read `ToEccentric(centre_of_mass)` as transforming the centre of mass into the eccentric frame.  Since it's a factory, make it clear in the name; since it's not called often, invest in a long name.  `MakeRigidToEccentricMotion` maybe?